docs/release-notes/8.2.asciidoc

```diff
@@ -2,14 +2,27 @@
 == 8.2
 
 [discrete]
-[[known-issue-8.2.0]]
-==== Known issues
-* On macOS versions before 12.4, if {elastic-endpoint} is used with other products that monitor or manage network traffic (such as antivirus programs, firewalls, or VPNs), users might experience network connection issues. To resolve this issue, upgrade to macOS 12.4 or later.
+[[release-notes-8.2.1]]
+=== 8.2.1
+
+[discrete]
+[[bug-fixes-8.2.1]]
+==== Bug fixes and enhancements
+* Allows {kibana-ref}/pre-configured-connectors.html[preconfigured connectors] to be used with cases ({pull}130372[#130372]).
+* Adds pagination to the *Table* tab on the Alert details flyout to fix a performance issue on the Timelines page ({pull}131358[#131358]).
+* Fixes sorting issues that were related to unmapped fields ({pull}132190[#132190]).
+* Fixes a bug in the *Filter In*, *Filter Out*, and *Add to timeline investigation* inline actions that caused incorrect results to be retrieved ({pull}132251[#132251]).
+* Enhances performance by improving calculations for the top count function and hover action. ({pull}131363[#131363]). 
```

Comment:
I think we should specify the feature here, otherwise it's pretty vague. 

```suggestion
* Enhances performance for Data views by improving calculations for the top count function and hover action. ({pull}131363[#131363]). 
```

Reply:
I don't think we should add this, the change fixes a problem that scaled with the number of table cells on the page that had hover actions, and for any hover action cell in the app, but was independent of data views.

Reply:
@jmikell821 I see your point about clarifying what gained a performance bump. From what I understand, this PR affects the "top n hover" action that handles inline actions and the the **Show top value** action. Here's one example of the "top n hover" action in the Alerts table. 


https://user-images.githubusercontent.com/79124755/169864402-b4c9a076-ad55-483a-86bd-cd31417ad66f.mp4


It's my understanding that the **Show top value** action will only display for certain fields as it's tied to specific data types. So when users hover over a field in a data grid, there's logic happening behind the scenes that determining whether to display the **Show top value** option. If the conditions are met, more is done to retrieve and display the appropriate values. Prior to this fix, all of this was a very expensive workflow that required lengthy checks and calculations across the entire Security app. With the fix, the "top n hover" action uses less resources and is overall, much smoother and faster user experience throughout the app. 

@kqualters-elastic feel free to correct anything that's wrong or add your own input to clarify!

Reply:
@kqualters-elastic pretty sure this action is available across all tables that display inline actions for values, right? @jmikell821 how about:

```suggestion
* Enhances performance by improving calculations for the top count function and hover action in data grids ({pull}131363[#131363]). 
```

Reply:
@nastasha-solomon `data tables` works for me. Thanks!